test/config/stack_config.json

```diff
@@ -64,6 +64,23 @@
       }
     }
   },
+  "docker_local_run":{
+    "rackhd_host": "localhost",
+    "type": "linux",
+    "ucsm_ip" : "172.31.128.252",
+    "ucs_service_uri" : "https://localhost:7080",
+    "ucsm_config_file" : "./tests/ucs/ucsm_config.xml",
+    "install-config": {
+      "ports": {
+        "mongo_port": 27017,
+        "http": 9090,
+        "https": 8443,
+        "amqp_ssl": 5672,
+        "amqp_nossl": 5671,
+        "ssh": 22
```

Comment:
Is this really the ssh-port we reach the docker container with? When "ssh -p 22 localhost" is done from the host executing the tests, you need to log into the _container_....